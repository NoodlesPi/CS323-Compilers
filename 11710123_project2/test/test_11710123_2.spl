// type 2
int main(){
    helloworld(1);
    return 1;
}