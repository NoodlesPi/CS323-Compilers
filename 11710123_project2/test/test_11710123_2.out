Error type 2 at Line 3: function is invoked without definition
